tests/attested_tls/run_scenario.sh

```diff
@@ -0,0 +1,78 @@
+#! /bin/bash
```

Comment:
Please see my comments on this file from Huseyin's PR #650.

Also suggest putting this file under `tests/attestation/common` so it can shared by both test cases.